feature/dashboard/src/test/java/com/simprints/feature/dashboard/settings/syncinfo/SyncInfoViewModelTest.kt

```diff
@@ -302,17 +290,15 @@ class SyncInfoViewModelTest {
         every { syncOrchestrator.observeImageSyncStatus() } returns MutableStateFlow(mockNotSyncingImageStatus)
         createViewModel()
         viewModel.isPreLogoutUpSync = false
-        val observer = mockk<Observer<LiveDataEventWithContent<Unit>>>(relaxed = true)
-        val slot = slot<LiveDataEventWithContent<Unit>>()
-        val capturedValues = mutableListOf<LiveDataEventWithContent<Unit>>()
-        every { observer.onChanged(capture(slot)) } answers {
-            capturedValues.add(slot.captured)
-        }
 
-        viewModel.logoutEventLiveData.observeForever(observer)
+        val flowCollector = async {
+            viewModel.logoutEventFlow.collect {
+                // fail if any logout event is emitted
+                assertThat(true).isFalse()
```

Comment:
Pretty sure there is a `fail("fail if any logout event is emitted")` method for that.